pycbc/inference/likelihood.py

```diff
@@ -545,6 +545,34 @@ def loglikelihood(self, **params):
         """
         return self._dist.logpdf([params[p] for p in self.variable_args])
 
+class TestEggbox(BaseLikelihoodEvaluator):
+    r"""The test distribution is an 'eggbox' function:
+    $$f(\Theta)=-\left[2+\prod_{i=1}^{n}\cos\left(\frac{\theta_{i}}{2}\right)\right]^{5}$$
```

Comment:
Could you use the `:math::` call out box here instead, and make it clear that this is what the log-likelihood is set to; i.e.:
```
r"""The test distribution is an 'eggbox' function:

.. math::

    \log \mathcal{L}(\Theta) = \left[2+\prod_{i=1}^{n}\cos\left(\frac{\theta_{i}}{2}\right)\right]^{5}

The number...
```